### ISSUE TITLE
api error using python 

### ISSUE THREAD

**Issue by alaaaali** [VideoGuys/python-client#5]
Hi,

i had an issue with using this api and i got the error 
{'code': 400, 'message': 'invalid api route method', 'errors': []}

here is the python code

import videoguys

api_client = videoguys.ApiClient(
  api_token="<api-token>", api_host="[vevio|vidup]"
)

video_info = api_client.getVideoInfo("v2rqq94nzvr6")
print(video_info)

Could anyone advise why i got the error and how to fix it

**Comment by VideoGuys** [comment]
Hi, the main issue with the code you've provided is the usage of the 'api_host' parameter. It must be either 'vevio' **xor** 'vidup'. That could resolve the issue you are facing.

On a more serious note, it is **highly** advised not to share your API token with anyone. I've gone ahead and edited it out for you.

I would suggest destroying and recreating the token you are using as it is likely compromised at this point.

**Comment by alaaaali** [comment]
Hi, 
i changed the parameter as you mention but it didn't work and still got the same error 

{'code': 400, 'message': 'invalid api route method', 'errors': []}
{'code': 400, 'message': 'invalid video specified', 'errors': []}

here is the code

api_client = videoguys.ApiClient(
  api_token="", api_host="vevio"
)

try:
    video_info = api_client.getVideoInfo("zl32wn9v7k3v")
    print(video_info)
    
    pair_response = api_client.getVideoPair("v2rqq94nzvr6")
    print(pair_response)
except:
    raise

**Comment by VideoGuys** [comment]
That error is caused when the video code is invalid, in this case the video you are specifying seems to be deleted.

If you try to view the video, you should get an error specifying the video has been deleted.

**Comment by alaaaali** [comment]
i don't think so as if you tried the video ID "v0oeqelxkwo6" you will got the same error even the video is available and watchable.
